alpaca_trade_api/stream.py

```diff
@@ -64,7 +64,7 @@ def __init__(self,
         }
         self._name = 'data'
         self._should_run = True
-        self._maxSymbolsInMessage = 100  # should be change
+        self._max_frame_size = 128  # should be change
```

Comment:
128 bytes is probably too little. I would put `32768`. But maybe ask @gnvk too 🤔